uint8 rs485_slave_id # 1-247
uint32 rs485_baudrate # 1200, 2400, 4800, 9600, 14400, 19200, 28800, 38400, 56000, 57600, 115200, 128000, 256000
uint8 rs485_parity # 0 = None, 1 = Odd, 2 = Even
